### DISCUSSION TITLE
Add preset modes for humidifier card

### DISCUSSION THREAD

**Discussion by Japing** [home-assistant/frontend#18884] [Dashboard] [answered]
* Add please preset modes for new design humidifier card as it is for climates, not only on/off.
* Add preset modes for humidifier in tile card.
*  Add set_humidify for humidifier in tile card.

**Comment by piitaya** [comment]
Preset modes doesn't exist for humidifier.
A slider to set the humidifier can be a good feature :)

**Reply by Japing** [reply]
why doesn't it exist?
available_modes on https://developers.home-assistant.io/docs/core/entity/humidifier/

![1](https://github.com/home-assistant/frontend/assets/12009800/1f2dcacf-07fd-4ce9-9999-c819b2c0a17c)

![2](https://github.com/home-assistant/frontend/assets/12009800/4894d477-c17c-496b-b76f-9f39abc45a65)

**Comment by piitaya** [comment]
Oh right, my bad, I forgot this one 😅

**Comment by piitaya** [comment, marked as answer]
Hi 👋

Target humidity is added : https://github.com/home-assistant/frontend/pull/18913
Old humidifier modes is renamed to humidifier toggle and new humidifier modes is added : https://github.com/home-assistant/frontend/pull/18912

**Reply by Japing** [reply]
Thank you very much.  :+1:

**Reply by Japing** [reply]
@piitaya Good work!
Now the target temperature for the thermostat in the tiles card is only available with buttons, but the target humidity with slider.
I have a suggestion: add buttons for target humidity in the tiles card (it is more practicable), or add the option to select slider or buttons (as is for the number).
Big thx.
![3](https://github.com/home-assistant/frontend/assets/12009800/b10d7176-49dc-4ca0-9cb5-a13b72e0b0e1)
